### PR TITLE
added eindino net

### PR DESCRIPTION
**Pull request by gamingcodedev**
## Type of change

- [ X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/eindino/manifest.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "server_id": 887294676128890900,
+    "rename_to_minecraft_name": true
+  },
+  "gamemodes": {
+    "citybuild": {
+      "name": "CityBuild",
+      "color": "#55FF00",
+      "versions": "1.20<*"
+    },
+    "SMP": {
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:31]: JSON key must be lowercase

File: minecraft_servers/eindino/manifest.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "web_shop": "https://shop.eindino.net",
+    "web_support": "https://forum.eindino.net",
+    "twitter": "einDinoNET",
+    "discord": "https://discord.eindino.net",
+    "teamspeak": "eindino.net",
+    "tiktok": "eindinonet"
+  },
+  "discord": {
+    "server_id": 887294676128890900,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:23]: Reserved for partners only; change to false

File: minecraft_servers/eindino/manifest.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://forum.eindino.net",
+    "web_shop": "https://shop.eindino.net",
+    "web_support": "https://forum.eindino.net",
+    "twitter": "einDinoNET",
+    "discord": "https://discord.eindino.net",
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:17]: Custom links are reserved for partners only; use a normal discord.gg/[INVITE] link instead

File: minecraft_servers/eindino/manifest.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "eindino.de",
+    "eindino.net"
+  ],
+  "supported_languages": [
+    "de", "en"
+  ],
+  "social": {
+    "web": "https://forum.eindino.net",
+    "web_shop": "https://shop.eindino.net",
+    "web_support": "https://forum.eindino.net",
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:15]: web_support cannot be a forum; must be a faq, contact page or similar

File: minecraft_servers/eindino/manifest.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{
+  "server_name": "eindino",
+  "nice_name": "einDino",
+  "direct_ip": "eindino.net",
+  "server_wildcards": [
+    "eindino.de",
+    "eindino.net"
+  ],
+  "supported_languages": [
+    "de", "en"
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:10]:
```suggestion
    "de",
    "en"
```

File: minecraft_servers/eindino/manifest.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
+{
+  "server_name": "eindino",
+  "nice_name": "einDino",
+  "direct_ip": "eindino.net",
+  "server_wildcards": [
```
Comment (maaaathis) [inline minecraft_servers/eindino/manifest.json:5]: Doesn’t look like a real wildcard 
IMG_1173
IMG_1174

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**